parent/pom.xml

```diff
@@ -243,11 +243,13 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
+						<include>**/*Test.class</include>
 					</includes>
+					<excludes>
+						<exclude>**/*Abstract*.class</exclude>
+						<exclude>**/$*.class</exclude>
```

Comment:
There is if you believe in good devdoc. https://github.com/jbosstools/jbosstools-build/pull/167/files#r23538008